assets/js/src/core/modules/asset/asset-api-slice.gen.ts

```diff
@@ -68,7 +68,7 @@ export type GetAssetsApiArg = {
     pathIncludeDescendants?: boolean;
 };
 export type GetAssetCustomSettingsByIdApiResponse = /** status 200 Array of custom settings */ {
-    customSettings?: CustomSettings;
+    items?: CustomSettings;
```

Comment:
Just to be sure this is change by the code generation, right?

Reply:
yes this was an issue from the backend - but i guess due to the extension of the API (metadata, workflows) it would be a good idea to merge these PRs and then check again (in this PR) if there are any additional changes - to have a clean state of the API.